spec/lib/rmt/cli/main_spec.rb

```diff
@@ -76,10 +84,14 @@
 
         before do
           allow(RMT::Lockfile).to receive(:create_file).and_raise(RMT::Lockfile::ExecutionLockedError)
+          allow(File).to receive(:read).with(RMT::Lockfile::LOCKFILE_LOCATION).and_return(pid)
         end
 
         it do
-          expect { command }.to output(/Process is locked\n/).to_stdout
+          expect(described_class).to receive(:exit)
+          expect { command }.to output(
+            "Process is locked by the application with pid #{pid}. Close this application or wait for it to finish before trying again\n"
+          ).to_stderr
```

Comment:
this it block is a 1:1 duplicate with the one above :arrow_right: shared example